cloud/deployment/dag_deployment.go

```diff
@@ -0,0 +1,41 @@
+package deployment
+
+import (
+	astro "github.com/astronomer/astro-cli/astro-client"
+	"github.com/pkg/errors"
+)
+
+func Initiate(deploymentID string, client astro.Client) (astro.InitiateDagDeployment, error) {
```

Comment:
please add docs line, since it's exported name. 
```suggestion
// Initiate dag deployment ... 
func Initiate(deploymentID string, client astro.Client) (astro.InitiateDagDeployment, error) {
```